application/src/main/webapp/vue-app/components/ExoChatContact.vue

```diff
@@ -173,9 +175,11 @@ export default {
     },
     contactAvatar() {
       if (this.type === 'u') {
-        return getUserAvatar(this.userName);
```

Comment:
shouldn't we remove old functions that retrieve avatars ?

Reply:
The Old function return return `${chatConstants.SOCIAL_USER_API}${user}/avatar`;
we need to retrieve the new avatars with the lastModified attribute /portal/rest/v1/social/users/root/avatar?lastModified=1630604606385&r=uz1dBGKQFq2OiTlaw89eErNF5P%2FsLsMtr0tZSYxhczgRlpmi1Wz6auFUgjOsMNs0
to update it dynamically in the chat application

Reply:
Are this function still useful ? if not, let's remove it 
 https://github.com/exoplatform/chat-application/blob/3887f8ee43a0bf7868d16b37116413b09d89322c/application/src/main/webapp/vue-app/chatServices.js#L384